### ISSUE TITLE
Translations Missing

### ISSUE THREAD

**Issue by ZeldaZach** [mtgjson/mtgjson#478]
Gotta see why Translations didn't populate last full build

**Comment by omfgitsmark** [comment]
Still not populating. Where are they pulled from?

**Comment by ZeldaZach** [comment]
Wizard's website. https://magic.wizards.com/en/products/card-set-archive 

Change "en" to whatever you need.

**Comment by omfgitsmark** [comment]
Hmm. I tested the soup parsing for a few languages and they worked fine. I think they updated the english version recently since the new sets (basically anything in standard) are in there own little thing at the top now, instead of the normal table design, and I don't remember it looking like that. I don't think that would affect building the translation table though, would it? You'd know better than I would.

My other suspicions are something with contextVars since it's kinda stored as one, and contextVars always like to be annoying, or maybe something to do with caching. I noticed it tries to build the table after every set because there are few "couldn't find SET icon" messages after each set, so I think it's not getting stored for whatever reason. I have a better understanding of how it works now so I can investigate/troubleshoot a little better while doing a build.
